docusaurus/vercel.json

```diff
@@ -127,22 +128,22 @@
     },
     {
       "source": "/cloud/getting-started-with-airbyte-cloud",
-      "destination": "/platform/using-airbyte/getting-started/",
+      "destination": "/platform/using-airbyte/getting-started",
       "statusCode": 301
     },
     {
       "source": "/quickstart/deploy-airbyte",
-      "destination": "/platform/using-airbyte/getting-started/",
+      "destination": "/platform/using-airbyte/getting-started",
```

Comment:
maybe this one should be: /platform/using-airbyte/getting-started/oss-quickstart